### PR TITLE
REFACTOR: old patterns and deprecations

### PR DESCRIPTION
**Pull request by jjaffeux**

### CODE REVIEW

File: assets/javascripts/discourse/widgets/code-review-commit-approved-notification-item.js.es6 [hunk 1]
```diff
@@ -3,16 +3,15 @@ import { DefaultNotificationItem } from "discourse/widgets/default-notification-
...
 replaceIcon("notification.code_review_commit_approved", "check");
 
 createWidgetFrom(
   DefaultNotificationItem,
   "code-review-commit-approved-notification-item",
   {
-    notificationTitle() {
-      return I18n.t("notifications.code_review.commit_approved.title");
-    },
+    notificationTitle: i18n("notifications.code_review.commit_approved.title"),
```
Comment (danielwaterworth) [inline assets/javascripts/discourse/widgets/code-review-commit-approved-notification-item.js.es6]: The `notificationTitle` isn't a property. It's consumed [here](https://github.com/discourse/discourse/blob/master/app/assets/javascripts/discourse/widgets/default-notification-item.js.es6#L111).

**Comment by danielwaterworth** [comment]
@jjaffeux, I appreciate you doing this. If you fix the `notificationTitle`, I'd be happy to see this merged.

File: assets/javascripts/discourse/widgets/code-review-commit-approved-notification-item.js.es6 (hunk 1 above)
Comment (jjaffeux) [reply assets/javascripts/discourse/widgets/code-review-commit-approved-notification-item.js.es6]: hmmm right

why the params though ?

```
const title = this.notificationTitle(notificationName, data);
```

clearly seems not used 🤔 will remove

Comment (jjaffeux) [reply assets/javascripts/discourse/widgets/code-review-commit-approved-notification-item.js.es6]: OH OK nevermind I see, it's from core

**Comment by jjaffeux** [comment]
> @jjaffeux, I appreciate you doing this. If you fix the `notificationTitle`, I'd be happy to see this merged.

Should be good to merge

**Comment by danielwaterworth** [comment]
I merged this on the command line to squash the commits, but retain @jjaffeux's authorship.

**Pull request closed without merging**